src/config/Config.ts

```diff
@@ -37,7 +37,7 @@ const integrationTypes: Record<IntegrationType, IntegrationTypeConfigBase> = {
         name: 'Email'
     },
     [IntegrationType.TEAMS]: {
-        name: 'Teams'
+        name: 'Microsoft Teams'
```

Comment:
@josejulio Is this the good place to fix so that the frontend shows `Microsoft Teams` instead of `Teams` while creating an integration?